cli/tests/integration_tests.rs

```diff
@@ -3489,45 +3489,27 @@ console.log("finish");
     output: "redirect_cache.out",
   });
 
-  itest!(deno_test_coverage {
-    args: "test --coverage --unstable test_coverage.ts",
-    output: "test_coverage.out",
+  itest!(test_coverage_branch {
+    args: "run --allow-all test_coverage_branch.ts",
+    output: "test_coverage_branch.out",
     exit_code: 0,
   });
 
-  itest!(deno_test_comment_coverage {
-    args: "test --coverage --unstable test_comment_coverage.ts",
-    output: "test_comment_coverage.out",
+  itest!(test_coverage_branch_lcov {
+    args: "run --allow-all test_coverage_branch_lcov.ts",
+    output: "test_coverage_branch_lcov.out",
     exit_code: 0,
   });
 
-  itest!(deno_test_branch_coverage {
-    args: "test --coverage --unstable test_branch_coverage.ts",
-    output: "test_branch_coverage.out",
+  itest!(test_coverage_complex {
+    args: "run --allow-all test_coverage_complex.ts",
+    output: "test_coverage_complex.out",
```

Comment:
Nitpick: please nest all coverage tests in `mod coverage` and put fixture files in `cli/tests/coverage`

Reply:
It's a bit odd since the working directory is in tests and itest doesn't let you change it but moved them.